api/schema/v1/modules/packetio.yaml

```diff
@@ -332,6 +337,7 @@ definitions:
             enum:
               - static
               - dhcp
+              - auto
```

Comment:
Added explicit configuration method for "auto" instead of using missing ipv4 interface to indicate it.
This allows for configuring IPv6 only stacks.
IPv6 config has same "auto" option for SLAAC.